### PR TITLE
Add whitepaper

### PR DESCRIPTION
**Pull request by hlb8122**
**Motivation**
Good overview of the project, the goals, the main components and how they interact together.

### CODE REVIEW

File: cashweb/whitepaper.tex [hunk 1]
```diff
@@ -253,23 +268,25 @@ \section{Applications}
...
 
 \subsection{Open Messaging}
 
-The combination of the POP protocol, keyservers, and relay servers, allows for advanced privacy features and SPAM-free communication. The ability to send structured messages with attached value, allows for all kinds of fee-based human-to-human, human-to-machine, machine-to-human, and machine-to-machine message processing. The most obvious application of this is peer to peer payments and messaging. However, the potential for other interesting services such as robots, which exchange value as well as information, can be imagined.
+The combination of the POP protocol, keyservers, and relay servers, allows for advanced privacy features and SPAM-free communication. The ability to send structured messages with attached value, allows for all kinds of fee-based human-to-human, human-to-machine, machine-to-human, and machine-to-machine message processing. The most obvious application of this is peer to peer payments and messaging. However, the potential for other interesting services such as online "bots", marketplaces, Web3.0 applications, and decentralized financial protocols.
```
Comment (rawb0t) [inline cashweb/whitepaper.tex:279]: >However, the potential for other interesting services such as online "bots", marketplaces, Web3.0 applications, and decentralized financial protocols.

This sentence isn't complete. Perhaps "there is potential" or something similar

File: cashweb/whitepaper.tex [hunk 2]
```diff
@@ -253,23 +268,25 @@ \section{Applications}
...
 
 \section{Conclusion}
 
-The CashWeb protocol's aim to provide censorship-resistant solutions to common technological problems which continually are reinvented in proprietary ways. CashWeb, like the underlying cryptocurrency technology, allows for the disintermediation of authentication, identity management, and messaging.
+The CashWeb protocol's aim to provide censorship-resistant solutions to common technological problems which continually are reinvented in proprietary ways. CashWeb, like the underlying cryptocurrency technology, allows for the disintermediation of authentication, identity management, and messaging. It enables the seemless use of payments across all internet infrastructure without the introduction of permissioned financial intermediaries.
```
Comment (rawb0t) [inline cashweb/whitepaper.tex:283]: >seemless 

SEAMLESS.

File: cashweb/whitepaper.tex [hunk 3]
```diff
@@ -253,23 +268,25 @@ \section{Applications}
 
 \subsection{Standardized Authorization \& Authentication}
 
-The Proof-Of-Payment (POP) protocol enables the standardized and seemless use of existing HTTP technology, but without the use of complicated billing infrastructure and account management. It allows the pseudonymously purchase of a JWT API token, without requiring an account management system, billing frontends, or other complicated infrastructure.
+The Proof-Of-Payment (POP) protocol enables the standardized and seemless use of existing HTTP technology, but without the use of complicated billing infrastructure and account management. It allows the pseudonymously purchase of a JWT\cite{rfc7519} API token, without requiring an account management system, billing frontends, or other complicated infrastructure.
```
[the remark this answers is not in the document]
Comment (schancel) [reply cashweb/whitepaper.tex:271]: Funny nobody caught the seemless. It's mispelled all over

File: cashweb/whitepaper.tex [hunk 4]
```diff
@@ -189,7 +189,7 @@ \subsubsection{Relay Servers}
   \item Relay servers serve only specific users and therefore can cheaply host large amounts of encrypted personal data.
 \end{itemize}
 
-Uploading to, and pulling messages from, the relay server should be protected by the Proof-of-Payment protocol. This allows for paid, but pseudonymous registration to the service; protecting user privacy. By using a standardized authentication and authorization mechanism, along with the global keyserver network, users can easily migrate between relay service providers should they choose.
+Uploading to, and pulling messages from, the relay server should be protected by the Proof-of-Payment protocol. This allows for paid, but pseudonymous registration to the service; protecting user privacy. Users can easily migrate between relay service providers should they choose due to using a standardized authentication and authorization mechanism, along with the global keyserver network.
```
Comment (rawb0t) [inline cashweb/whitepaper.tex:192]: >service; protecting

probably use a comma rather than a semicolon

File: cashweb/whitepaper.tex [hunk 5]
```diff
@@ -171,19 +171,19 @@ \subsubsection{Keyservers}
...
 
 \subsubsection{Relay Servers}
 
-Relay servers provide the combined purpose of both POP and SMTP servers. They accept messages on behalf of the clients, and verify basic integrity of these messages. They also host profile information including avatars and other information. While this server currently only provide messaging, profile names, and icons, they can easily be extended to provide status messages, microblogs, and other potentially useful functions.
+Relay servers provide the combined purpose of both POP and SMTP servers. They accept messages on behalf of the clients, and verify basic integrity of these messages. They also host profile information including avatars and other information. While the relay servers currently only provides messaging, profile names, and icons, the software can easily be extended to provide status messages, microblogs, and other potentially useful functions.
```
Comment (rawb0t) [inline cashweb/whitepaper.tex:184]: should have left 'provide' the way it is

**Pull request merged**